app/models.py

```diff
@@ -46,10 +46,11 @@ class Hospital(db.Model):
     zipcode = db.Column(db.String(128), default="")
     name = db.Column(db.String(140), default="")
     credit = db.Column(db.Integer, default=0)
-    wants = db.relationship('Wants', backref='author1', lazy='dynamic')
-    has = db.relationship('Has', backref='author2', lazy='dynamic')
-    users = db.relationship('User', backref='author3', lazy='dynamic')
-    exchange = db.relationship('Exchange', backref='author7', lazy='dynamic')
+    wants = db.relationship('Wants', backref='hospital')
```

Comment:
This allows you to, for example, do `Wants.query.first().hospital` and get a `Hospital` object.